src/ray/raylet_client/raylet_client.h

```diff
@@ -187,6 +187,8 @@ class RayletClientInterface : public PinObjectsInterface,
   virtual void GetGcsServerAddress(
       const rpc::ClientCallback<rpc::GetGcsServerAddressReply> &callback) = 0;
 
+  virtual void GCSRestart(const rpc::ClientCallback<rpc::GCSRestartReply> &callback) = 0;
```

Comment:
Similar to the comment on RPC method name, I think something like `NotifyGCSRestarted()` is clearer.